src/Sylius/Bundle/TaxonomyBundle/Form/Type/TaxonChoiceType.php

```diff
@@ -107,4 +100,25 @@ public function getBlockPrefix()
     {
         return 'sylius_taxon_choice';
     }
+
+    /**
+     * @param null|string   $root
+     * @param null|callable $filter
+     *
+     * @return array|TaxonInterface[]
```

Comment:
`TaxonInterface[]` would be sufficient, `array|` indicates as there could be something else ;)